Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -156,7 +160,7 @@ mkDriverRideRes rideDetails driverNumber rideRating (ride, booking) = do
       tripEndTime = ride.tripEndTime,
       rideRating = rideRating <&> (.ratingValue),
       chargeableDistance = ride.chargeableDistance,
-      exoPhone = booking.providerExoPhone
+      exoPhone = maybe booking.primaryExophone (\exophone -> if not exophone.isPrimaryDown then exophone.primaryPhone else exophone.backupPhone) mbExophone
```

Comment:
Instead of renaming booking.providerExophone to `primaryExophone` let's move this logic at booking level itself and keep the column name to `providerExophone`

Reply:
Same for rider-app too.